scrapers/karatmedia.yml

```diff
@@ -0,0 +1,45 @@
+name: karatmedia
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 18tokyo.com/set/
+      - alljapanesepass.com/set/
+      - analnippon.com/set/
+      - bigtitstokyo.com/set/
+      - bukkakenow.com/set/
+      - idols69.com/set/
+      - japaneseflashers.com/set/
+      - japaneseslurp.com/set/
+      - jcosplay.com/set/
+      - jpmilfs.com/set/
+      - jpnurse.com/set/
+      - jpshavers.com/set/
+      - jpteacher.com/set/
+      - jschoolgirls.com/set/
+      - myracequeens.com/set/
+      - ocreampies.com/set/
+      - officesexjp.com/set/
+      - outdoorjp.com/set/
+      - povjp.com/set/
+      - tokyobang.com/set/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class='b-box-title']
+      Date:
+        selector: //div[@class='b-likes-group-info']/div[2]
+        postProcess:
+          - parseDate: 02 January 2006
+      Details: //p[@class="b-video-info__text"]/text()
+      Tags:
+        Name: //p[@class="b-video-info__text"]//a
+      Studio:
+        Name:
+          selector: //span[@class="b-header-logo"]/text()
+      Image: //div[@class="b-fake-player"]//@src
+      Performers:
+       Name: //p/a[1]/span[1]
```

Comment:
For performers i prefer to not use generics eg Amateur/Japanese AV Model and leave the field blank instead.
If you want to do that you can use the following
```yaml
        Name: //p/a[1]/span[not(text()="Amateur" or text()="Japanese AV Model")]
```
If you prefer to use the `Amateur`  or `Japanese AV Model` one then  leave yours as it is now ( `span[1]` probably isnt needed, `span` should be enough) and add as a comment so that users can uncomment if needed
```yaml
          #Name: //p/a[1]/span[not(text()="Amateur" or text()="Japanese AV Model")] # Uncomment if you dont want Amateur,JAV Model
```
```
```